### ISSUE TITLE
File Transfer Speed

### ISSUE THREAD

**Issue by Dynasty-Dev** [racka98/Lenovo-Thinkpad-T450-T450s-Hackintosh-Guide-Opencore#56]
**Describe the bug**
I’ve noticed that file transfer speeds are slower than on windows

**To Reproduce**
Transfer any file to a USB or try to extract an archive

**Expected behavior**
I expected it to be as fast as on windows

**Screenshots**
N/A
**macOS Info:**
 - OS: Big Sur
 - Version: 11.5.2
 - Laptop Model: T450 (i5-5300|HD5500)

**Additional context**
N/A

**Comment by Dynasty-Dev** [comment]
Also in bios > USB can I set USB 3.0 to enabled instead of auto

**Comment by racka98** [comment]
You can try forcing it to always 3.0. Also you can check if it's working in 3.0 by using Hackintool

**Comment by Dynasty-Dev** [comment]
Yes also I was wondering why Bluetooth wasn’t working when I updated the kext! I figured it out, it wasn’t working because in the latest version of IntelBluetoothInjector they removed contents/macos folder so just delete that from executablepath in config.plist
